config/manager/manager.yaml

```diff
@@ -27,7 +27,7 @@ spec:
         - /manager
         args:
         - --enable-leader-election
-        image: controller:latest
+        image: xridge/kubestone:latest
```

Comment:
Then we should also change the IMG in the Makefile.

Reply:
good point, addressed it.